### PR TITLE
[Backport] Fixed product advanced pricing design issue

### PR DESCRIPTION
**Pull request by mage2pratik**
### Description (*)

Fixed product advanced pricing design issue
Admin >> Add new product >> Advance pricing >> In customer group price all fields alignment not proper.

### Fixed Issues (if relevant)

1. magento/magento2#18775: Product Advanced Pricing design issue

### Manual testing scenarios (*)

1. Go to Admin
2. Add new product
3. Advance pricing
4. See the Customer Group Price fields
5. Customer Group Price > Add new options

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

Backport for: https://github.com/magento/magento2/pull/18790

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @mage2pratik. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.2-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by mage2pratik** [comment]
@magento-engcom-team give me 2.2-develop instance

**Comment by magento-engcom-team** [comment]
Hi @mage2pratik. Thank you for your request. I'm working on Magento 2.2-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @mage2pratik, here is your [Magento instance](https://i-21229-2-2-develop.instances.magento-community.engineering).
Admin access: [https://i-21229-2-2-develop.instances.magento-community.engineering/admin](https://i-21229-2-2-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

File: app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less [hunk 1]
```diff
@@ -527,7 +527,6 @@
 
             & > .admin__field-label {
                 #mix-grid .column(@field-label-grid__column, @field-grid__columns);
-                background: @color-white;
```
Comment (sidolov) [inline app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less:530]: Why do we need to change the background color of all admin form fields to fix only one specific issue?

**Review by sidolov — CHANGES_REQUESTED** [review]

File: app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less (hunk 1 above)
Comment (mage2pratik) [reply app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less:530]: Background color is not require there and "background: @color-white;" is deleted in 2.3.x. [link](https://github.com/magento/magento2/blob/1fb83b185b66f161dfa55744c23e213887f53fcd/app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less#L547)

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-4311](https://jira.corp.magento.com/browse/ENGCOM-4311) has been created to process this Pull Request

**Pull request merged**